community_supplied/Subscriber-Service/service.aaa-monitor-radius-server-delay.rule

```diff
@@ -0,0 +1,159 @@
+healthbot {
+    topic service.aaa {
+        rule monitor-radius-server-delay {
+            keys server-address;
+            synopsis "Monitors RADIUS server delay";
+            description "Monitors RADIUS server delay from the BNG's perspective against a provided baseline";
+            sensor radius-server-information {
+                synopsis "Telemetry for RADIUS server response-time measurement";
+                description "Telemetry for RADIUS server response-time measurement";
+                open-config {
+                    sensor-name /junos/system/subscriber-management/aaa/radius-servers/radius-server;
+                    frequency 30s;
```

Comment:
See if frequency can be changed to 60s in all rules. It is just recommendation by keeping performance in mind